src/main/java/com/sendgrid/Client.java

```diff
@@ -195,15 +167,11 @@ public Response getResponse(CloseableHttpResponse response) throws IOException {
 	 * @return the response object
 	 */
 	public Response get(Request request) throws URISyntaxException, IOException {
-		URI uri = null;
-		HttpGet httpGet = null;
+		URI uri;
+		HttpGet httpGet;
 
-		try {
-			uri = buildUri(request.getBaseUri(), request.getEndpoint(), request.getQueryParams());
-			httpGet = new HttpGet(uri.toString());
-		} catch (URISyntaxException ex) {
-			throw ex;
-		}
+		uri = buildUri(request.getBaseUri(), request.getEndpoint(), request.getQueryParams());
+		httpGet = new HttpGet(uri.toString());
```

Comment:
You can even move the declaration and assignment to the same line, to merge line 170 with 173. The same is valid for the other methods: post, patch, put below.. etc.

Reply:
Thanks, fixed!